contrib/remote-dev-tool/conf/exports.template

```diff
@@ -0,0 +1 @@
+/workspace *(rw,fsid=0,sync,no_root_squash,no_subtree_check,anonuid=1000,anongid=1000)
```

Comment:
is this file needed?

Reply:
This file is needed for the NFS docker in ubuntu.